### PR TITLE
update for sentry

### PR DESCRIPTION
**Pull request by Crypto-One-dev**

### CODE REVIEW

File: src/pools/index.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 const BigNumber = require('bignumber.js')
+const express = require('express')
+const app = express()
```
Comment (Cryo45) [inline src/pools/index.js:3]: why re-creating the app here? does not make sense

File: src/runtime/app.js [hunk 2]
```diff
@@ -6,20 +6,28 @@ module.exports = function () {
   const { PORT, CORS_SETTINGS } = require('../lib/constants')
 
   const app = express()
+  const logger = require('../lib/logger/index')
   const cors = require('cors')
   const { startPollers } = require('./pollers')
   const initDb = require('../lib/db')
+  const sentry = logger()
```
Comment (Cryo45) [inline src/runtime/app.js:13]: this creates a new instance in each file. is it better to create only one instance inside `'../lib/logger/index'`?

**Comment by Cryo45** [comment]
i am afraid we can not merge it right now. there is a concern that this can leak ip and other sensitive information. let's hold this off.

**Comment by Crypto-One-dev** [comment]
ok

**Pull request closed without merging**